src/problems/DustDampedGyromotion/plot_dust_damped_gyromotion.py

```
@@ -0,0 +1,219 @@
+    return (
+        Line2D([], [], color="black", linestyle="--", label="analytic"),
+        Line2D([], [], color="C1", marker="s", markerfacecolor="none", linestyle="None", label="frozen"),
+        Line2D([], [], color="C0", marker="o", markerfacecolor="none", linestyle="None", label="iterated"),
+    )
+
+
+def make_figure(data_dir: Path, output_dir: Path) -> Path:
+    fig, axes = plt.subplots(1, 3, figsize=(DOUBLE_COLUMN_WIDTH, 2.55), sharey=True)
+    fig.subplots_adjust(left=0.08, right=0.99, bottom=0.16, top=0.88, wspace=0.17)
+
+    for ax, (case_tag, title, xlabel, xmax) in zip(axes, CASES):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)